### ISSUE TITLE
Don't capitalise FallbackStyles in core-blocks/paragraph

### ISSUE THREAD

**Issue by tofumatt** [WordPress/gutenberg#7355]
Capitalising the `FallbackStyles` HOC/variable in `core-blocks/paragraph/index.js` makes it seem like it's a component and is confusing to read–as pointed out here: https://github.com/WordPress/gutenberg/pull/6782/files#r191892085

We should lowercase it in https://github.com/WordPress/gutenberg/blob/master/core-blocks/paragraph/index.js#L43, and anywhere else it's used so it doesn't confusingly read as a component.

**Comment by gziolo** [comment]
If we want to follow recurring patterns, it should be renamed to withFallbackStyles.

**Comment by tofumatt** [comment]
True, though I think the function that is used to create the variable currently named `FallbackStyles` is already named `withFallbackStyles`, so maybe that'll need to be imported as something else 😄

**Comment by slushman** [comment]
Can this issue be closed? It appears there is only one other usage of "FallbackStyles" and it doesn't appear in the Paragraph block as referenced here.

**Comment by tofumatt** [comment]
It's in https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/button/edit.js#L31 too; I've changed the issue name to reflect my general dislike of the variable name over its naming in that specific file 😄
